src/components/study-container.jsx

```diff
@@ -729,22 +724,18 @@ export function StudyContainer({ view, onChangeTab }) {
         }
         // A modification has been added to the currentNode and this one has been built incrementally.
         // No need to load the network because reloadImpactedSubstationsEquipments will be executed in the notification useEffect.
-        if (
-            previousCurrentRootNetwork === currentRootNetworkUuid &&
-            isSameNode(previousCurrentNode, currentNode) &&
-            isNodeBuilt(previousCurrentNode)
-        ) {
```

Comment:
This is an error,
If currentRootNetwork changes then we must reset equipments in spreadsheets.

Reply:
OK! I reverted condition but using previous value in ref as in other useEffect